code/hdb/hdbstandard.q

```diff
@@ -1,7 +1,26 @@
 // reload function
 reload:{
+	
+	if[.z.w in key .hdb.reloadcalls;
+		.hdb.reloadcalls[.z.w]:1b;
+		.lg.o[`reload;"reload call received from handle ", string[.z.w], "; reload calls pending from handles ", ", "sv string where not .hdb.reloadcalls];
+		if[not all .hdb.reloadcalls;:(::)]];
 	.lg.o[`reload;"reloading HDB"];
+	@[`.hdb.reloadcalls;key .hdb.reloadcalls;:;0b];
 	system"l ."}
 
 // Get the relevant HDB attributes
 .proc.getattributes:{`date`tables!(@[value;`date;`date$()];tables[])}
+
+\d .hdb
+
+// dictionary of handles to reload
+reloadcalls:()!();
+
+// function to add handle to reloadcalls dictionary
+po:{[h] if[.z.u in `wdb`rdb;reloadcalls[h]:0b]};
```

Comment:
I think hardcoding the usernames here is probably the wrong way to go - ideally I think we should probably do this based on proctype, and we should have a config variable for which proctypes (or usernames, if we can't use proctypes for some reason) we need to keep track of (users of TorQ can change proctypes, and the internal usernames used by them etc.)

Reason for using proctypes rather than usernames is that I don't think elsewhere in TorQ usernames have any specific "meaning" and typically any process identification logic is based off proctype - if we do change that here, we have to make sure we document that very clearly & signpost users to the config variable they have to update if they're using different usernames.